routes/profile.js

```diff
@@ -2,6 +2,6 @@ const router = require('express').Router();
 const userController = require('../controllers/usersController');
  
 router.get('/', userController.getProfile); // read profile
-router.get('/edit', userController.update); // update profile
+router.get('/edit', userController.updatePage); // update profile
```

Comment:
unclear function name. try to make more descriptive names that reveals the true intention of the code.

updatePage is vague because you might ask "what page?"
whereas, updateProfile is a straightforward name ("ohh this code should update the profile") BUT it doesn't do that exactly.  It renders the editProfile page, it doesn't update anything.

Point: The code works (that's great 💯 ) but it's confusing for people who are reading the code (and for future maintenance). If we can make our code clean, we can make our code base easier to maintain and understand :)

Reply:
same comment for logUser